silkworm/db/snapshots/bittorrent/web_session.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+   Copyright 2024 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "web_session.hpp"
+
+#include <map>
+
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/asio/ssl/error.hpp>
+#include <boost/asio/ssl/stream.hpp>
+#include <boost/beast/core/flat_buffer.hpp>
+#include <boost/beast/core/tcp_stream.hpp>
+#include <boost/beast/ssl.hpp>
+#include <boost/beast/version.hpp>
+#include <boost/url/parse.hpp>
+
+#include <silkworm/infra/common/log.hpp>
+
+#include "root_certificates.hpp"
+
+namespace silkworm::snapshots::bittorrent {
+
+namespace net = boost::asio;
+namespace ssl = net::ssl;
+namespace urls = boost::urls;
+
+//! Custom HTTP headers to include in any request to web servers hosted by Cloudflare
+const std::map<std::string_view, std::string_view> kCloudflareHeaders{
+    {"lsjdjwcush6jbnjj3jnjscoscisoc5s", "I%OSJDNFKE783DDHHJD873EFSIVNI7384R78SSJBJBCCJBC32JABBJCBJK45"},
```

Comment:
where does this value come from?

Reply:
It comes from Erigon [here](https://github.com/erigontech/erigon/blob/main/erigon-lib/downloader/downloader.go#L154)